data/cmdvartab

```diff
@@ -47,6 +47,11 @@ VARDESC input.voltage "Input voltage (V)"
 VARDESC input.voltage.extended "Extended input voltage range"
 VARDESC input.voltage.maximum "Maximum incoming voltage seen (V)"
 VARDESC input.voltage.minimum "Minimum incoming voltage seen (V)"
+VARDESC input.voltage.status "Voltage status relative to the thresholds"
```

Comment:
not for voltage.status, but indeed for the thresholds (low, high).
Also applicable to docs/nut-names.txt.
Fixed in both files, thanks